contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -3892,7 +3896,7 @@ bbf_pivot(PG_FUNCTION_ARGS)
 	MemoryContextSwitchTo(oldcontext);
 
 	oldcontext = MemoryContextSwitchTo(tsql_outmost_context);
-	tsql_outmost_estat->pivot_parsetree_list = list_delete_nth_cell(tsql_outmost_estat->pivot_parsetree_list, tsql_outmost_estat->pivot_number - 1);
+	tsql_outmost_estat->pivot_parsetree_list = list_delete_nth_cell(tsql_outmost_estat->pivot_parsetree_list, 0);
```

Comment:
What does change to 0 in here mean?

Reply:
The execution order is first one in the list get executed first and that's why we change from last to 0. After execution, we will remove the head element of the list.